components/feature/tabs/src/test/java/mozilla/components/feature/tabs/WindowFeatureTest.kt

```diff
@@ -30,6 +31,7 @@ import org.mockito.Mockito.spy
 import org.mockito.Mockito.verify
 
 @RunWith(AndroidJUnit4::class)
+@FlowPreview
```

Comment:
Can we control this via a compiler argument (like `ExperimentalCoroutinesApi`) in build.gradle? This will avoid that this annotation ripples up all the way through app code.

Reply:
Ah yes, good point. I've added this to `build.gradle` now!